<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>227</green>
        <blue>203</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>110</green>
        <blue>175</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>136</red>
        <green>136</green>
        <blue>136</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>227</green>
        <blue>203</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>227</green>
        <blue>203</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>131</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>227</green>
        <blue>203</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>227</green>
        <blue>203</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>85</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>227</green>
        <blue>203</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>110</green>
        <blue>175</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>227</green>
        <blue>203</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>227</green>
        <blue>203</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>198</y>
      <width>1941</width>
      <height>786</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="ncbutton">
     <property name="geometry">
      <rect>
       <x>553</x>
       <y>121</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Next Customer</string>
     </property>
     <property name="autoRepeatDelay">
      <number>301</number>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ttcbutton">
     <property name="geometry">
      <rect>
       <x>743</x>
       <y>121</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ticket Type Creation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="garmpricingbutton">
     <property name="geometry">
      <rect>
       <x>943</x>
       <y>121</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Garment Pricing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gandccbutton">
     <property name="geometry">
      <rect>
       <x>1133</x>
       <y>121</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Garment + Color Creation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tocbutton">
     <property name="geometry">
      <rect>
       <x>1343</x>
       <y>121</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ticket Options Creation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createempbutton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>271</y>
       <width>130</width>
       <height>49</height>
      </rect>
     </property>
     <property name="text">
      <string>Create/Edit Employees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="managedeliveriesbutton">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>268</y>
       <width>143</width>
       <height>43</height>
      </rect>
     </property>
     <property name="text">
      <string>Manage Deliveries</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reportsbutton">
     <property name="geometry">
      <rect>
       <x>1159</x>
       <y>265</y>
       <width>142</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Reports</string>
     </property>
    </widget>
    <widget class="QPushButton" name="billingbutton">
     <property name="geometry">
      <rect>
       <x>749</x>
       <y>271</y>
       <width>137</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>Billing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="emailcustbutton">
     <property name="geometry">
      <rect>
       <x>1346</x>
       <y>263</y>
       <width>146</width>
       <height>43</height>
      </rect>
     </property>
     <property name="text">
      <string>Email Customers</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="bannerframe">
    <property name="geometry">
     <rect>
      <x>-32</x>
      <y>8</y>
      <width>1932</width>
      <height>190</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>139</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>89</red>
          <green>205</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>72</red>
          <green>172</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>28</red>
          <green>69</green>
          <blue>87</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>37</red>
          <green>93</green>
          <blue>117</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>65</red>
          <green>65</green>
          <blue>65</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>139</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>155</red>
          <green>197</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Accent">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>139</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>89</red>
          <green>205</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>72</red>
          <green>172</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>28</red>
          <green>69</green>
          <blue>87</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>37</red>
          <green>93</green>
          <blue>117</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>65</red>
          <green>65</green>
          <blue>65</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>139</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>155</red>
          <green>197</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Accent">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>28</red>
          <green>69</green>
          <blue>87</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>139</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>89</red>
          <green>205</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>72</red>
          <green>172</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>28</red>
          <green>69</green>
          <blue>87</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>37</red>
          <green>93</green>
          <blue>117</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>28</red>
          <green>69</green>
          <blue>87</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>28</red>
          <green>69</green>
          <blue>87</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>139</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>139</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>139</green>
          <blue>175</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>28</red>
          <green>69</green>
          <blue>87</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Accent">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>73</red>
          <green>181</green>
          <blue>227</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="chooseimage">
     <property name="geometry">
      <rect>
       <x>719</x>
       <y>2</y>
       <width>566</width>
       <height>184</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::WheelFocus</enum>
     </property>
     <property name="toolTipDuration">
      <number>-2</number>
     </property>
     <property name="locale">
      <locale language="Ewe" country="Ghana"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/images/Logo2.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="hanger1">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>63</y>
       <width>190</width>
       <height>183</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../PyFiles2/resources/hanger3.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="hanger2">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>57</y>
       <width>201</width>
       <height>187</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../PyFiles2/resources/hanger4.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="hanger3">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>-2</y>
       <width>201</width>
       <height>187</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../PyFiles2/resources/hanger1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="hanger4">
     <property name="geometry">
      <rect>
       <x>523</x>
       <y>4</y>
       <width>186</width>
       <height>184</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../PyFiles2/resources/hanger2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="hanger8">
     <property name="geometry">
      <rect>
       <x>1809</x>
       <y>5</y>
       <width>186</width>
       <height>184</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../PyFiles2/resources/hanger2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="hanger6">
     <property name="geometry">
      <rect>
       <x>1407</x>
       <y>62</y>
       <width>201</width>
       <height>187</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../PyFiles2/resources/hanger4.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="hanger5">
     <property name="geometry">
      <rect>
       <x>1305</x>
       <y>66</y>
       <width>190</width>
       <height>183</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../PyFiles2/resources/hanger3.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="hanger7">
     <property name="geometry">
      <rect>
       <x>1576</x>
       <y>4</y>
       <width>201</width>
       <height>187</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../PyFiles2/resources/hanger1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Image">
   <property name="text">
    <string>Add Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../PyFiles2/resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
